src/routes/SafeAppLandingPage/SafeAppLandingPage.tsx

```diff
@@ -0,0 +1,312 @@
+import { ReactElement, useCallback, useEffect } from 'react'
+import { useSelector } from 'react-redux'
+import { useLocation } from 'react-router-dom'
+import styled from 'styled-components'
+import { Card, Title, Button, Text, Loader } from '@gnosis.pm/safe-react-components'
+import Divider from '@material-ui/core/Divider'
+import ClickAwayListener from '@material-ui/core/ClickAwayListener'
+import List from '@material-ui/core/List'
+import Popper from '@material-ui/core/Popper'
+
+import SuccessSvg from 'src/assets/icons/safe-created.svg'
+import DemoSvg from 'src/assets/icons/demo.svg'
+import { getChainById, isValidChainId } from 'src/config'
+import { demoSafeRoute, history, OPEN_SAFE_ROUTE, WELCOME_ROUTE } from 'src/routes/routes'
+import { useAppList } from 'src/routes/safe/components/Apps/hooks/appList/useAppList'
+import { SafeApp } from 'src/routes/safe/components/Apps/types'
+import { getAppInfoFromUrl } from 'src/routes/safe/components/Apps/utils'
+import { userAccountSelector } from 'src/logic/wallets/store/selectors'
+import { useStateHandler } from 'src/logic/hooks/useStateHandler'
+import { setChainId } from 'src/logic/config/utils'
+import ProviderDisconnected from 'src/components/AppLayout/Header/components/ProviderInfo/ProviderDisconnected'
+import Provider from 'src/components/AppLayout/Header/components/Provider'
+import ConnectDetails from 'src/components/AppLayout/Header/components/ProviderDetails/ConnectDetails'
+import NetworkLabel from 'src/components/NetworkLabel/NetworkLabel'
+import Img from 'src/components/layout/Img'
+import Link from 'src/components/layout/Link'
+import { black300, grey400, secondary } from 'src/theme/variables'
+import useAsync from 'src/logic/hooks/useAsync'
```

Comment:
Like I mentioned earlier, these 28 imports would be much more manageable if the components were in separate modules/files.

Same goes for the styled components at the bottom of the file, which would be then nicely scoped to their corresponding components.